pages/resources/index.js

```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
     return a.kicker.localeCompare(b.kicker);
   });
 
+  const wrapper = await getProps(collections, recents);
   const navigation = await getNavigation();
 
   return {
     props: {
-      collections,
       navigation,
+      wrapper,
     },
   };
 };
+
+async function getProps(collections, recents) {
+  // create a wrapper which contains all necessary data for the page
+  // includes page content based on section (content) and headings to pass to the TOC
+  // we have to hard-code What's New because it's too unique
+  let wrapper = { content: {}, headings: [] };
+  let content = { categories: [] };
+  let [name, posts, description, size, type, id] = [
+    "What's New?",
+    recents,
+    null,
+    "full",
+    "New",
+    "What's%20New?",
+  ];
+  let category = { name, posts, description, size, type, id };
```

Comment:
It looks like you're using category only inside the loop. I would move these declarations inside the loop since that's you don't need them to persist outside of that.